python/pyspark/ml/wrapper.py

```diff
@@ -87,9 +87,19 @@ def _new_java_array(pylist, java_class):
           - bool -> sc._gateway.jvm.java.lang.Boolean
         """
```

Comment:
We should better update `_new_java_array`'s document to reflect this change.

Reply:
Just a gentle ping on doing this part